requirements.txt

```diff
@@ -1,3 +1,4 @@
+pylint
```

Comment:
This file lists only the runtime dependencies of the addon-checker. `pylint` isn't a runtime dependency.

Reply:
Then I will add this in travis script.

Reply:
It's back again.